cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2547,6 +2547,57 @@ action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365` sou
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 
+[O365 - Failed Login From Unusual Country]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Failed O365 Login From Unusual Country. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -2h@h
+dispatch.latest_time = +2h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m  Workload=AzureActiveDirectory Operation=UserLoginFailed  `cs_public_ips(ClientIP)` \
+| fields _time sourcetype user ClientIP Id LogonError user_type authentication_method app ApplicationId \
+| append \
+    [| search `cs_o365` sourcetype=o365:graph:api source="AuditLogs.SignIns" earliest=-2h@m latest=now  status.errorCode=0 `cs_public_ips(ipAddress)` \
+    | fields _time sourcetype userPrincipalName ipAddress id LogonError appDisplayName status.failureReason clientAppUsed conditionalAccessStatus isInteractive deviceDetail.* org_country org_region org_city \
```

Comment:
For (status.failReason) when I run the query it doesn't show data .. I tried several things to make it work but it doesn't show any data so can you please check the query maybe you can recognize if I'm missing something